cliqz/package.json

```diff
@@ -10,7 +10,7 @@
     "jsc-android": "^236355.1.1"
   },
   "dependencies": {
-    "browser-core": "https://s3.amazonaws.com/cdncliqz/update/edge/rn-ghostery-android/master/1.36.0.1b42.7f571e3.tgz",
+    "browser-core": "https://s3.amazonaws.com/cdncliqz/update/edge/rn-ghostery-android/master/1.36.0.1b60.296ffce.tgz",
```

Comment:
@khaled-cliqz Shall we cherry-pick this commit on 1.35 branch and point to it?

Reply:
@giota-cliqz, no, when we cherry-pick everything, we'll then make another PR with the release branch